packages/host/app/components/operator-mode/stack-item.gts

```diff
@@ -69,20 +70,22 @@ export default class OperatorModeStackItem extends Component<Signature> {
     fieldType: FieldType | undefined;
   }>();
 
-  get renderedLinksToCards(): RenderedLinksToCard[] {
+  get renderedCardsForOverlayActions(): RenderedCardForOverlayActions[] {
     return (
       this.cardTracker.elements
         .filter((entry) => {
           return (
             entry.meta.format === 'data' ||
             entry.meta.fieldType === 'linksTo' ||
-            entry.meta.fieldType === 'linksToMany'
+            entry.meta.fieldType === 'linksToMany' ||
+            entry.meta.fieldType === 'contains'
```

Comment:
"contains one" card now has a clickable overlay too (it can be opened in the stack)

Reply:
It looks like we'll probably include `containsMany` in this list too, which means all of our field types. Seems like maybe we don't need this `fieldType` property anymore. In `field-component`, we're only rendering overlays in the`isCard` case, which covers all cards.

Reply:
Yeah, good observation.